lib/rubocop/cop/style/for.rb

```diff
@@ -38,6 +38,10 @@ module Style
       #     end
       #   end
       #
+      # @safety
+      # This cop's autocorrection is unsafe because the scope of
+      # variables is different between `each` and `for`.
```

Comment:
Can you indent it?

```suggestion
      # @safety
      #   This cop's autocorrection is unsafe because the scope of
      #   variables is different between `each` and `for`.
```